### PR TITLE
Travis setup for gradle build

### PR DESCRIPTION
**Pull request by heidijinxujia**
- Update the code from dev

### CODE REVIEW

**Comment by heidijinxujia** [comment]
Need to find a way to shorten the time of travis CI, maybe in the travis configuration or in the gradle build configuration.

**Comment by imakhalova** [comment]
Cool, when are we planning to switch?)

**Comment by heidijinxujia** [comment]
Before switching to AS, we need to fix all the static code analysis issues from Lint, PMD and findbugs.

File: adal/build.gradle [hunk 1]
```diff
@@ -33,7 +33,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```
Comment (RandalliLama) [inline adal/build.gradle]: @heidijinxujia Why?

**Comment by weijjia** [comment]
@heidijinxujia the travis set up looks good! As we talked today, re-branch from dev, and copy over all the gradle changes along with all the travis changes to the new branch. Sounds good to you? :)

**Pull request closed without merging**